sleap/gui/widgets/video.py

```diff
@@ -485,6 +485,8 @@ def showInstances(self, show):
         """
         for inst in self.instances:
             inst.showInstances(show)
+        for inst in self.predicted_instances:
+            inst.showInstances(show)
```

Comment:
Test coverage for this line might be good given the mess related to `LabeledFrame.instances_to_show` mentioned above.

Also, in light of #518, perhaps the tests should check that the color of the `QtInstance` objects don't change? I think the colors are determined when they're instantiated, so toggling the visibility might not affect it, but it would be good to check.

If the test for color is unnecessary after looking into it, that's fine.

Reply:
Test coverage has been added, but is skipped for ubuntu.